src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -83,13 +133,26 @@ const SendTilBeslutterFooter: React.FC<{
         <>
             {behandlingErRedigerbar && (
                 <Footer>
+                    {kanOppretteFremleggsoppgave && (
+                        <Alertstripe variant={'info'} fullWidth={true}>
+                            Det blir automatisk opprettet oppgave for kontroll av
+                            inntektsopplysninger 1 år frem i tid når denne behandlingen blir
+                            godkjent av beslutter{' '}
+                            <Checkbox
```

Comment:
Hvordan unngå at denne skfiter linje ?

Reply:
Skal det ikke være en ny rad for hver checkbox? Fra skissene:
<img width="634" alt="image" src="https://user-images.githubusercontent.com/937168/226859418-a15754b1-e62d-452b-9038-26c035efa389.png">

Reply:
For å få tekst og checkbox på samme rad kan du wrappe teksten og checkboksen inn i en flexbox, for eksempel en div med verdiene display: flex; gap: 1rem; (mulig du må prøve deg frem på antall rems)